cdc/model/changefeed.go

```diff
@@ -58,9 +72,10 @@ type ChangeFeedInfo struct {
 	Engine       SortEngine   `json:"sort-engine"`
 	SortDir      string       `json:"sort-dir"`
 
-	Config *config.ReplicaConfig `json:"config"`
-	State  FeedState             `json:"state"`
-	Error  *RunningError         `json:"error"`
+	Config   *config.ReplicaConfig `json:"config"`
+	State    FeedState             `json:"state"`
+	ErrorHis []int64               `json:"history"`
```

Comment:
how about using `time.Duration` instead of `int64`?

Reply:
This filed will be stored into etcd, and may be shared by different captures (as the owner can be transferred), so int64 is used here to store timestamp.